### DISCUSSION TITLE
Error updating Magic Mirror

### DISCUSSION THREAD

**Discussion by JanWillemDeBoer** [tteck/Proxmox#2036] [Help Needed]
While updating Magic Mirror I get this error, does anyone have advice on how to solve this issue?

<img width="692" alt="image" src="https://github.com/tteck/Proxmox/assets/150567333/268d17b8-46d2-4d1c-b5e4-5c6f8ac5c24c">

**Comment by JanWillemDeBoer** [comment]
@tteck Do you have any ideas? I can't get it solved unfortunately, thanks 👍

**Reply by tteck** [reply]
I can't replicate the issue.

```
    __  ___            _      __  ____                     
   /  |/  /___ _____ _(_)____/  |/  (_)_____________  _____
  / /|_/ / __  / __  / / ___/ /|_/ / / ___/ ___/ __ \/ ___/
 / /  / / /_/ / /_/ / / /__/ /  / / / /  / /  / /_/ / /    
/_/  /_/\__,_/\__, /_/\___/_/  /_/_/_/  /_/   \____/_/     
             /____/                                     
 
 ✓ Updated Successfully
root@magicmirror:~# 
```

**Reply by JanWillemDeBoer** [reply]
Is there any extra information that might help you to solve it?

**Reply by tteck** [reply]
If I cannot reproduce the problem on my side, I cannot address an issue that appears to be functioning correctly here.

**Reply by tteck** [reply]
You can try throwing more resources to it.

**Reply by JanWillemDeBoer** [reply]
Didn't help unfortunately, will try to do a reinstall, thanks 👍
